lib/computer_player.rb

```diff
@@ -21,7 +21,6 @@ def place_ship(boat)
       placement = @board.cells.keys.shuffle[0..(boat.length - 1)]
       @board.valid_placement?(boat, placement)
     end
-    # require "pry"; binding.pry
     @board.place(boat, placement)
```

Comment:
Nice job on remembering to remove the pry haha

Reply:
important but pesky to remember to remove lol